velox/vector/DictionaryVector.h

```diff
@@ -176,6 +176,17 @@ class DictionaryVector : public SimpleVector<T> {
   void addNulls(const uint64_t* bits, const SelectivityVector& rows) override {
     flatNullsBuffer_ = nullptr;
     BaseVector::addNulls(bits, rows);
+
+    // Resize indices_ to be as large as nullsBuffer.
+    auto bufferPtr = mutableIndices(rows.end());
```

Comment:
Are we guaranteed that rows.end() >= length_? Probably not. Consider taking a max(rows.end(), length_).

naming: bufferPtr -> indices

Reply:
Thanks Masha, Good catch.